src/librustdoc/config.rs

```diff
@@ -229,6 +229,22 @@ impl fmt::Debug for Options {
     }
 }
 
+#[derive(Clone, Debug)]
+pub(crate) enum PathOrURL {
```

Comment:
```suggestion
pub(crate) enum PathOrUrl {
```
to follow idiom, and to be consistent with the style in this crate